src/models.py

```diff
@@ -583,7 +583,9 @@ def build_single_sentence_module(task, d_inp: int, use_bert: bool, params: Param
         - SingleClassifier (nn.Module): single-sentence classifier consisting of
```

Comment:
Header for this fn. says that pool_type depends on use_bert. Is that still right?